ocs_ci/ocs/constants.py

```diff
@@ -1015,3 +1015,6 @@
 # Minimum cluster requirements in term of node specs
 MIN_NODE_CPU = 16
 MIN_NODE_MEMORY = 64 * 10 ** 9
+
+# aws tags
+AWS_CLOUDFORMATION_TAG = 'aws:cloudformation:stack-name'
```

Comment:
I think that you also need to move the reference to this constant in ocs_ci/cleanup/aws/cleanup.py:188

Reply:
I deleted the reference in **ocs_ci/cleanup/aws/defaults.py** and used the one in **ocs_ci/ocs/constants.py**

Reply:
thanks!